cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -88,16 +92,29 @@ class ReporterCommand : CliktCommand(
         .file(mustExist = false, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = false)
         .required()
 
-    private val reporters by option(
+    private val reportFormats by option(
         "--report-formats", "-f",
         help = "The list of report formats that will be generated."
-    ).convert { reporterName ->
-        // Map upper-cased reporter names to their instances.
-        val reporters = Reporter.ALL.associateBy { it.reporterName.toUpperCase() }
-        reporters[reporterName.toUpperCase()]
-            ?: throw BadParameterValue("Reporters must be contained in ${reporters.keys}.")
+    ).convert { format ->
+        reporters[format.toUpperCase()]
+            ?: throw BadParameterValue("Report formats must be one or more of ${reporters.keys}.")
     }.split(",").required()
 
+    private val reportOptions by option(
+        "--report-option", "-O",
+        help = "Specify a report-format-specific option to be passed to the reporter implementation. The key is the" +
```

Comment:
How about omitting " to be passed to the reporter implementation" as it is redundant, because command line options are always passed to the actual implementation.

Reply:
That's not correct, there are at least some command line options, like e.g. `outputDir`, that are not passed to the reporter implementation at all.

Reply:
I probably wasn't clear enough, I didn't mean to say that all command line options do get passed the a reporter instance. What I meant was that it suffices to say that "Specify a report-format-specific option.", and that "to be passed to the reporter implementation" does not really matter as it is implementation detail and IMO obvious.

Reply:
Ok, done.